shell/packages/sandstorm-ui-applist/applist-client.js

```diff
@@ -182,6 +185,13 @@ Template.sandstormAppList.events({
       launchAndEnterGrainByActionId(actionId);
     });
   },
+  "click .uninstall-action": function(event) {
+    var actionId = this._id;
+    var appId = UserActions.findOne(actionId).appId;
+    console.log("Uninstalling action " + actionId);
```

Comment:
Didn't really care one way or the other, since it's a single line triggered by a user action.  But I'll remove it, since it probably adds little value.